src/courseware/course/sidebar/common/SidebarBase.jsx

```diff
@@ -63,17 +145,24 @@ const SidebarBase = ({
       {showTitleBar && (
         <>
           <div className="d-flex align-items-center">
-            <span className="p-2.5 d-inline-block">{title}</span>
+            <h2
+              className="p-2.5 d-inline-block"
+              style={{
+                fontSize: '1.125rem', fontWeight: '400', color: '#454545', margin: '0', lineHeight: '1.5556',
```

Comment:
[question] Is it OK to hardcode styles? This may cause additional problems in the future when we will use Design Tokens.

Reply:
Yeah, lets do this 💯

Reply:
We also can use paragon classes instead of declaration it in style files: 
- `margin: '0'` style => `m-0` class
- `color: #454545;` style => `text-gray-700` class

Reply:
Corrected